### PR TITLE
Fix CPU governor checks.

### LINKED ISSUE #19: Linux CPU governor is not being checked.
I was inspecting bencher3 prior to running benchmarks. I've found that if you put a breakpoint in the Linux implementation of  check_cpu_throttled(), it is never hit. Subsequently, benchmarking is allowed to continue powersave mode, which is bad.

### LINKED ISSUE #20: Linux throttle check only looks at one cpu.
`CPU_GOV_FILE = "/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor"`

Bencher3 has cpu1, cpu2, and cpu3 in addition. We should check these too.

### PR DESCRIPTION
**Pull request by vext01**
* Fix missing governor check.
 * Check all CPUs, not just the first.

Fix  #19. Fix #20.

OK?

### CODE REVIEW

File: krun/platform.py [hunk 1]
```diff
@@ -155,13 +160,22 @@ def has_cpu_cooled(self):
                 return (False, reason)  # one or more sensor too hot
         return (True, None)
 
-    def check_cpu_throttled(self):
-        with open(LinuxPlatform.CPU_GOV_FILE, "r") as fh:
-            v = fh.read().strip()
+    def check_cpus_throttled(self):
+        """Checks the Linux CPU governors for the cpus.
+
+        Since we do not know which CPU benchmarks will be sceduled on,
```
Comment (ltratt) [inline krun/platform.py]: Typo.

**Comment by vext01** [comment]
fixed.

**Comment by vext01** [comment]
ready for rereview

**Comment by ltratt** [comment]
Once the typo commit is squashed, this should be OK to merge.

**Comment by vext01** [comment]
The commit has been squashed.

**Pull request merged**